core/dbt/cli/main.py

```diff
@@ -608,7 +608,7 @@ def retry(ctx, **kwargs):
     task = RetryTask(
         ctx.obj["flags"],
         ctx.obj["runtime_config"],
-        ctx.obj["manifest"],
```

Comment:
Is it okay for the manifest to be `None`? Would it be safer to do something like `ctx.obj.get("manifest")` which returns either the object or `None`?

Reply:
So the last refactor actually refactored parsing for retry into the task itself. So it would be alright to always pass in a None(which is what our intergration tests where doing before)

Reply:
We don't even use the manifest passed to the RetryTask's constructor at all at all: https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/task/retry.py#L66

This should be safe to entirely remove, we don't even need to pass `None`

Reply:
Done!

Reply:
Awesome! Thank y'all for the clarification 🙂

Reply:
Thanks for poking and resulting in better comment.